File: Cpp/basic.cpp
#include <iostream>
#include "task.h"
#include "vm.h"

int main()
{
    Vm newvm = Vm();
    Task taskOne = Task();
    taskOne.mips=1000;
    Task taskTwo = Task();
    taskTwo.mips = 1000;
    newvm.assaignTask(taskOne);
    newvm.assaignTask(taskTwo);
    newvm.run();
}

//TODO CreateTask, CreateVm functions maybe usefull

File: Java/Basic.java
public class Basic {
    public static void main(String[] args) {
        Vm newvm = new Vm();
        Task taskOne = new Task();
        taskOne.mips = 1000;
        Task taskTwo = new Task();
        taskTwo.mips = 1000;
        newvm.assaignTask(taskOne);
        newvm.assaignTask(taskTwo);
        newvm.run();
    }   
}
